mlora.py

```diff
@@ -112,23 +123,39 @@ def load_base_model(config: Dict[str, any]) -> Tuple[mlora.Tokenizer, mlora.LLMM
     return tokenizer, model
 
 
-def init_lora_model(config: Dict[str, any], llm_model: mlora.LLMModel):
+def init_lora_model(config: Dict[str, any], llm_model):
     if args.disable_lora:
         return
 
     for lora_config in config["lora"]:
         lora_weight = None
-        if args.load_lora:
-            adapter_file_path = lora_config["output"] + "/adapter_model.bin"
-            print(f"load {adapter_file_path}")
-            lora_weight = torch.load(adapter_file_path)
+        if "mixlora" in config and config["mixlora"]:
+            if args.load_lora:
```

Comment:
increase the cyclomatic complexity. maybe just use the args.load_moe

Reply:
moe is different with lora, we can just split it to different mode;

Reply:
use unified `load_from_checkpoint`? This moe is a little different from the typical moe. It is like a mixture of loras.